policyengine_us/variables/gov/states/dc/dhs/gac/eligibility/dc_gac_eligible_child.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class dc_gac_eligible_child(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = "Eligible child for DC General Assistance for Children (GAC) based on demographics"
+    reference = (
+        "https://code.dccouncil.gov/us/dc/council/code/sections/4-205.05a"
+    )
+    defined_for = StateCode.DC
```

Comment:
etc - do consistently
```suggestion
    defined_for = "dc_pap_eligible_child"
```